supplementary_style_guide/style_guidelines/general-formatting.adoc

```diff
@@ -10,8 +10,52 @@ Follow _The IBM Style Guide_  guidance of _day Month year_ for date formats, for
 When the format _day Month year_ causes a presentation or clarity issue, use instead: _Month day, year_ (for example, October 3, 2019).
 
 
-[[user-replaced-variables]]
-== User-replaced variables
+[[user-replaced-values]]
+== User-replaced values
+
+A _user-replaced value_, or _replaceable value_, is a value that must be replaced by the user with a value that is relevant for their situation. User-replaced values are often found in places such as code blocks, file paths, and commands.
+
+User-replaced values should use a descriptive names, and should follow this general format: _<value_name>_.
+
+Replaceable values should have the following characteristics:
```

Comment:
To replace the 'should' to be more direct per IBM, you could change this to something like one of these options:

- Use the following format for user-replaced values:
- Replaceable values must have the following characteristics:
- Ensure that user-replaced values have following characteristics:

It might also be nicer to keep referring to them as 'user-replaced values' for consistency throughout the body of the entry?